great-idea/css/index.css

```diff
@@ -62,3 +62,82 @@ h1, h2, h3, h4, h5 {
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+/* Nav Bar */
+header {
+	margin-bottom: 50px;
+	line-height: 75px;
+}
+
+header nav {
+	display: inline-block;
+}
+
+header nav a {
+	text-decoration: none;
+	color: grey;
+	margin: auto 35px;
+	font-size: 12px;
+}
+
+.logo {
+	margin: 20px 0px -10px 0px;
+	display: inline-block;
+}
+
+/*For Innocation on Demand and Code Globe */
+.cta {
+	margin-bottom: 100px;
+}
+
+.cta:after {
```

Comment:
I really like the unique way you employed a pseudo-element here to implement this change even though it wasn't specifically covered in class.  Great job!